### PR TITLE
Add firebase-admin

### LINKED ISSUE #720: Allow notifications in the application
This will eventually allow notifications in the application to keep the users engaged and in sync with the latest updates in the organization.
- Use [firebase-admin](https://www.npmjs.com/package/firebase-admin) package to implement notifications in the backend.
- Save FCM token on login.
- New mutation for logout to clear the FCM token.
- Create notification on new event.

### PR DESCRIPTION
**Pull request by guptapriyanshu7**
New mutation for logout
Create notification on new event

**What kind of change does this PR introduce?**

feature

**Issue Number:**

Fixes #720 

**Does this PR introduce a breaking change?**

No.

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md).**
Yes

### CODE REVIEW

**Comment by palisadoes** [comment]
@TheHazeEffect @JamaicanFriedChicken please review.

File: lib/resolvers/auth_mutations/saveFcmToken.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const { NotFoundError } = require('errors');
+const requestContext = require('talawa-request-context');
+
+module.exports = async (parent, args, context) => {
+  const user = await User.findById(context.userId);
+  if (!user) {
+    throw new NotFoundError(
+      requestContext.translate('user.notFound'),
+      'user.notFound',
+      'user'
```
Comment (JamaicanFriedChicken) [inline lib/resolvers/auth_mutations/saveFcmToken.js:11]: hey @guptapriyanshu7, swap these strings with the ones from the constants file ~

**Review by JamaicanFriedChicken — CHANGES_REQUESTED** [review]

File: lib/resolvers/auth_mutations/saveFcmToken.js (hunk 1 above)
Comment (guptapriyanshu7) [reply lib/resolvers/auth_mutations/saveFcmToken.js]: Fixed.

**Review by JamaicanFriedChicken — APPROVED** [review]
Looks good to me now~ @palisadoes you can merge this in

**Review by palisadoes — APPROVED** [review]

**Pull request merged**